core/src/test/scala/unit/kafka/log/LogTest.scala

```diff
@@ -1798,7 +1798,7 @@ class LogTest {
     }
 
     // reopen the log
-    log = createLog(logDir, logConfig, recoveryPoint = 200L)
+    log = createLog(logDir, logConfig)
```

Comment:
Is this change intended? The default value for  recoveryPoint is not 200.

Reply:
Yes. After we get rid of the sanity check, the index files for segment below the recovery point will no longer get checked on startup so in order to test rebuild a corrupted index, we need to make sure the recovery point is below the base offset for the corrupted index.